src/main/java/kr/allcll/seatfinder/seat/SeatStorage.java

```diff
@@ -21,6 +22,19 @@ public List<Seat> getNonMajorSeats(int limit) {
             .toList();
     }
 
+    public List<Seat> getPinSeats(List<Subject> subjects) {
```

Comment:
요기 pin이라는 이름을 붙여서 재사용하기 어려운 메서드가 되었어요. 지우면 어떨까요? 

```suggestion
    public List<Seat> getSeats(List<Subject> subjects) {
```